ui/src/app/views/workflowv3/run/workflowv3-run.component.ts

```diff
@@ -0,0 +1,145 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AutoUnsubscribe } from 'app/shared/decorator/autoUnsubscribe';
+import { Tab } from 'app/shared/tabs/tabs.component';
+import { Observable } from 'rxjs';
+import { finalize } from 'rxjs/operators';
+import { GraphDirection } from '../graph/workflowv3-graph.lib';
+import { WorkflowV3StagesGraphComponent } from '../graph/workflowv3-stages-graph.component';
+import { JobRun, WorkflowRunV3 } from '../workflowv3.model';
+
+@Component({
+    selector: 'app-workflowv3-run',
+    templateUrl: './workflowv3-run.html',
+    styleUrls: ['./workflowv3-run.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+@AutoUnsubscribe()
+export class WorkflowV3RunComponent implements OnInit, OnDestroy {
+    @ViewChild('graph') graph: WorkflowV3StagesGraphComponent;
+
+    data: WorkflowRunV3;
+    direction: GraphDirection = GraphDirection.VERTICAL;
+    resizing = false;
+    loading = false;
+    errors: Array<{ jobName: string, stepNumber: number }> = [];
+    problems: Array<string> = [];
+    infos: Array<string> = [];
+    projectKey: string;
+    tabs: Array<Tab>;
+    selectedTab: Tab;
+    selectJobRun: JobRun;
+
+    constructor(
+        private _cd: ChangeDetectorRef,
+        private _http: HttpClient,
+        private _activatedRoute: ActivatedRoute
+    ) {
+        this.tabs = [<Tab>{
+            translate: 'common_problems',
+            icon: 'exclamation triangle',
+            key: 'problems',
+            default: true
+        }, <Tab>{
+            translate: 'common_infos',
+            icon: 'exclamation circle',
+            key: 'infos'
+        }, <Tab>{
+            translate: 'common_results',
+            icon: 'list',
+            key: 'results'
+        }];
+    }
+
+    ngOnDestroy(): void { } // Should be set to use @AutoUnsubscribe with AOT
+
+    ngOnInit(): void {
+        const parentParams = this._activatedRoute.snapshot.parent.params;
+        const params = this._activatedRoute.snapshot.params;
+        this.projectKey = parentParams['key'];
+        const workflowName = parentParams['workflowName'];
+        const runNumber = params['number'];
+
```

Comment:
Be carreful, it won't work if wen can navigate between 2 runs.
Maybe subscribe to the route

Reply:
For now it's not possible to navigate between runs, to do when adding a new runs menu.